src/sentry/api/serializers/rest_framework/sentry_app.py

```diff
@@ -42,14 +42,25 @@ def validate(self, data):
             raise ValidationError(e.message)
 
 
+class URLField(serializers.URLField):
+    def validate(self, url):
+        # The Django URLField doesn't distinguish between different types of
+        # invalid URLs, so do any manual checks here to give the User a better
+        # error message.
+        if not url.startswith('http'):
+            raise ValidationError('URL must start with http[s]://')
```

Comment:
can we add a test for an invalid url? Maybe for the `webhook_url` since that's required.